src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -117,25 +127,49 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
 
 </Alert>
 
-### Troubleshooting
+<Alert level="note">
+
+If you use Okta to assign organization-level roles, you’ll be unable to make membership changes through Sentry and will need to continue using Okta.
```

Comment:
For users that are assigned a Sentry organization-level role through Okta, you'll be unable to make role changes to them through Sentry. You will have to edit the role in their app user profile in Okta and sync them to Sentry.

Reply:
syncing doesn't do anything at the moment. If they absolutely need to change roles, they will need to deprovision and reprovision the user.